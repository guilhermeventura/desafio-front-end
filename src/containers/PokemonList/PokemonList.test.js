import React from "react";
import { Provider } from "react-redux";
import configureMockStore from "redux-mock-store";
import thunk from "redux-thunk";
import renderer from "react-test-renderer";

import { default as PokemonList } from "./index";

describe("<PokemonList />", () => {
  const middlewares = [thunk];
  const mockStore = configureMockStore(middlewares);
  const store = mockStore({ pokemons: [] });
  it("renders correctly", () => {
    const component = renderer
      .create(
        <Provider store={store}>
          <PokemonList />
        </Provider>
      )
      .toJSON();

    expect(component).toMatchSnapshot();
  });
});
