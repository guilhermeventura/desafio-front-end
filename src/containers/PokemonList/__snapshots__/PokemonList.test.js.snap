// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PokemonList /> renders correctly 1`] = `
Array [
  <div
    className="columns is-multiline is-centered"
    style={Object {}}
  >
    <div
      className="column"
      style={Object {}}
    >
      <h3
        className="mdc-typography mdc-typography--headline3"
      >
        Your Pokedex is empty
      </h3>
    </div>
  </div>,
  <div
    className="columns is-multiline is-centered"
    style={Object {}}
  />,
]
`;
